### PR TITLE
Fixing Spark 3.2,3.3 pushdowns

### PR DESCRIPTION
**Pull request by davidrabinowitz**

### CODE REVIEW

**Review by VinayLondhe14 — APPROVED** [review]
Can we please add JoinExtractorSuite and BinaryOperationExtractorSuite unit tests to `spark-3.2-bigquery-pushdown_2.12` module?

File: pom.xml [hunk 1]
```diff
@@ -154,6 +156,7 @@
         <module>spark-bigquery-pushdown/spark-bigquery-pushdown-common_2.12</module>
         <module>spark-bigquery-pushdown/spark-2.4-bigquery-pushdown_2.12</module>
         <module>spark-bigquery-pushdown/spark-3.1-bigquery-pushdown_2.12</module>
+        <module>spark-bigquery-pushdown/spark-3.2-bigquery-pushdown_2.12</module>
```
[the remark this answers is not in the document]
Comment (davidrabinowitz) [reply pom.xml:159]: We don't have Spark 3.3 DSv2 connector

**Comment by davidrabinowitz** [comment]
> Can we please add JoinExtractorSuite and BinaryOperationExtractorSuite unit tests to spark-3.2-bigquery-pushdown_2.12 module?

Done

File: pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (VinayLondhe14) [reply pom.xml]: Screen Shot 2022-10-04 at 6 20 51 PM

Then should we remove the checkmark for Spark 3.3 here?

**Pull request merged**